pyvips/tests/test_foreign.py

```diff
@@ -68,8 +79,6 @@ def save_load_file(self, format, options, im, thresh):
         self.assertTrue(max_diff <= thresh)
         x = None
 
-        os.unlink(filename)
```

Comment:
Removed all `os.unlink` because the folder is getting deleted when it's done testing this class.